isic_challenge_scoring/classification.py

```diff
@@ -63,24 +63,29 @@ def __init__(
 
         self.overall = self.aggregate.at['balanced_accuracy']
 
-        if validation_metric == 'balanced_accuracy':
-            self.validation = metrics.balanced_multiclass_accuracy(
-                truth_probabilities, prediction_probabilities, truth_weights.validation_weight
-            )
-        elif validation_metric == 'ap':
-            self.validation = metrics.average_precision(
-                truth_probabilities, prediction_probabilities, truth_weights.validation_weight
-            )
-        elif validation_metric == 'auc':
-            self.validation = metrics.auc(
-                truth_probabilities, prediction_probabilities, truth_weights.validation_weight
-            )
+        if validation_metric:
```

Comment:
Should update the type annotation / initialization to allow `self.validation` to be optional.